b2sdk/b2http.py

```diff
@@ -236,6 +236,9 @@ class B2Http(object):
 
     """
 
+    # timeout for HTTP GET/POST requests
+    TIMEOUT = 120
```

Comment:
in case of a server-side issue, setting it at exactly 120s will cause the client to raise a timeout exception a few milliseconds before we receive a FIN packet from the server. It will cause all timeout errors to appear to be client-side and thus will suggest a client-side issue, while in reality most of those issues will actually be caused by the server. It would be nice for this to be easily recognizeable by the stacktrace in most of the cases, to only reserve the client-side timeout for a very misbehaving server.
Therefore please increase the timeout by at least 5 seconds (as heavily overloaded networks frequently have 4+s latency).